float32 speed
float32 distance_m
float32 distance_km
int32 current_position
int32 distance_mm
bool standstill
bool direction_forward
bool direction_backward
